server/models.py

```
@@ -110,4 +110,34 @@ def decrypt_entry(encrypted_data: dict, master_password: str) -> dict:
             'ciphertext': encrypted_data['ciphertext']
         }
         
-        return VaultCrypto.decrypt_data(encrypted_content, key)
\ No newline at end of file
+        return VaultCrypto.decrypt_data(encrypted_content, key)
```

Ruff (0.8.2)

Undefined name `VaultCrypto`

(F821)